logserver/src/tests/java/alluxio/logserver/AlluxioLog4jSocketNodeTest.java

```diff
@@ -0,0 +1,251 @@
+    }
+
+    @Test
+    public void testLevel()throws IOException, ClassNotFoundException {
+        Level level = Level.DEBUG;
+        createOutputStreams();
+        objectOutputStream.writeObject(level);
+        write2BufferAndCloseOutputStream();
+        createInputStreamsAndsetAcceptList();
+        Level level1 = (Level) validatingObjectInputStream.readObject();
+    }
+
+    @Test
+    public void testUtilLoggingLevel()throws IOException, ClassNotFoundException {
+        UtilLoggingLevel utilLoggingLevel = UtilLoggingLevel.INFO;
+        createOutputStreams();
+        objectOutputStream.writeObject(utilLoggingLevel);
+        write2BufferAndCloseOutputStream();
+        createInputStreamsAndsetAcceptList();
+        UtilLoggingLevel utilLoggingLevel1 =
+                (UtilLoggingLevel) validatingObjectInputStream.readObject();
+    }
+
+    @Test
+    public void testThrowable()throws IOException, ClassNotFoundException {
+        Throwable throwable = new Throwable();
+        createOutputStreams();
+        objectOutputStream.writeObject(throwable);
+        write2BufferAndCloseOutputStream();
+        createInputStreamsAndsetAcceptList();
+        Throwable throwable1 = (Throwable) validatingObjectInputStream.readObject();
+    }
+
+    @Test
+    public void testLogLevelFormatException()throws IOException, ClassNotFoundException {
+        LogLevelFormatException logLevelFormatException = new LogLevelFormatException("message");
+        createOutputStreams();
+        objectOutputStream.writeObject(logLevelFormatException);
+        write2BufferAndCloseOutputStream();
+        createInputStreamsAndsetAcceptList();
+        LogLevelFormatException logLevelFormatException1 =
+                (LogLevelFormatException) validatingObjectInputStream.readObject();
+    }
+
+    @Test
+    public void testLogTableColumnFormatException()throws IOException, ClassNotFoundException {
+        LogTableColumnFormatException logTableColumnFormatException =
+                new LogTableColumnFormatException("message");
+        createOutputStreams();
+        objectOutputStream.writeObject(logTableColumnFormatException);
+        write2BufferAndCloseOutputStream();
+        createInputStreamsAndsetAcceptList();
+        LogTableColumnFormatException logTableColumnFormatException1 =
+                (LogTableColumnFormatException) validatingObjectInputStream.readObject();
+    }
+
+    @Test
+    public void testPropertySetterException()throws IOException, ClassNotFoundException {
+        PropertySetterException propertySetterException = new PropertySetterException("string");
+        createOutputStreams();
+        objectOutputStream.writeObject(propertySetterException);
+        write2BufferAndCloseOutputStream();
+        createInputStreamsAndsetAcceptList();
+        PropertySetterException propertySetterException1 =
+                (PropertySetterException) validatingObjectInputStream.readObject();
+    }
```

Comment:
These tests fail with `InvalidClassException: Class name not accepted`

If this is intended behavior, assert an expected exception with `assertThrows`: https://www.baeldung.com/junit-assert-exception